projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/monitoring/DataJobMetrics.java

```diff
@@ -173,6 +172,33 @@ public void updateTerminationStatusGauge(final DataJob dataJob) {
         }
     }
 
+    /**
+     * Creates a "taurus.datajob.termination.status.counter" counter for the specified
+     * data job and execution status, if one does not exist. If a counter already
+     * exists, its value is incremented.
+     *
+     * @param dataJob The data job for which to increment the counter.
+     * */
+    public void incrementTerminationStatusCounter(final DataJob dataJob) {
+        Objects.requireNonNull(dataJob);
+
```

Comment:
Also from PR description

> However, if we want to aggregate data for all data jobs that we have deployed, for
> example check what percentage of all jobs fail with user or platform error compared
> to all job executions, things can get complicated.

Are the other data job status metrics usable prior this change? 
Is the reason for the change is like "introducing one additional metric that is easier to use yet not that informative"?
Is such a counter applicable for monitoring & data jobs troubleshooting? What is the use-case?

Reply:
I think a good answer to that question is to give an example query of how it would be used. We need it for our amon dashboards

Reply:
An example query would be
```
sum(increase(data_jobs_termination_status_counter{status=2}[24h]  / sum(increase(data_jobs_termination_status_counter[24h]))
```